src/app/user/registration/_components/owner-form.tsx

```diff
@@ -60,7 +60,8 @@ export default function OwnerForm(props: {
     const { userEmail } = form.getValues();
     form.clearErrors("emailOtp");
     if (isValidEmail(userEmail)) {
-      await sendLoginOtp({ emailAddress: userEmail, accountType: null });
+      // TODO: if the result of postOtpRequest is FAILED, we should ask user to request for another.
+      await postOtpRequest({ emailAddress: userEmail, accountType: null });
```

Comment:
looks important. if this fails, the user will be waiting a long time for their otp. we should inform them. make a ticket for this.

Reply:
Ticket created.